### PR TITLE
Update Google Cloud Storage API URL

### PR DESCRIPTION
**Pull request by matteosilv**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart Version bumped
- [X] CHANGELOG.md updated
- [X] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

No variables have been changed, only constant values in templates

### CODE REVIEW

File: stable/artifactory-ha/templates/artifactory-binarystore.yaml [hunk 1]
```diff
@@ -125,8 +125,8 @@ data:
 
         <provider id="google-storage" type="google-storage">
             <providerId>google-cloud-storage</providerId>
-            <endpoint>commondatastorage.googleapis.com</endpoint>
-            <httpsOnly>false</httpsOnly>
+            <endpoint>storage.googleapis.com</endpoint>
+            <httpsOnly>true</httpsOnly>
```
Comment (eldada) [inline stable/artifactory-ha/templates/artifactory-binarystore.yaml]: Can you make `httpsOnly` a parameter in `values.yaml`?

File: stable/artifactory/templates/artifactory-binarystore.yaml [hunk 2]
```diff
@@ -41,8 +41,8 @@ data:
 
         <provider id="google-storage" type="google-storage">
             <providerId>google-cloud-storage</providerId>
-            <endpoint>commondatastorage.googleapis.com</endpoint>
-            <httpsOnly>false</httpsOnly>
+            <endpoint>storage.googleapis.com</endpoint>
+            <httpsOnly>true</httpsOnly>
```
Comment (eldada) [inline stable/artifactory/templates/artifactory-binarystore.yaml]: Can you make `httpsOnly` a parameter in `values.yaml`?

File: stable/artifactory-ha/templates/artifactory-binarystore.yaml (hunk 1 above)
Comment (matteosilv) [reply stable/artifactory-ha/templates/artifactory-binarystore.yaml]: sure. what do you think about making also endpoint a parameter defaulting to "storage.googleapis.com" ?

File: stable/artifactory/templates/artifactory-binarystore.yaml (hunk 2 above)
Comment (matteosilv) [reply stable/artifactory/templates/artifactory-binarystore.yaml]: sure. what do you think about making also endpoint a parameter defaulting to "storage.googleapis.com" ?

File: stable/artifactory-ha/templates/artifactory-binarystore.yaml (hunk 1 above)
Comment (eldada) [reply stable/artifactory-ha/templates/artifactory-binarystore.yaml]: Good idea. Go ahead and add it. We'll merge after that.

File: stable/artifactory/templates/artifactory-binarystore.yaml (hunk 2 above)
Comment (eldada) [reply stable/artifactory/templates/artifactory-binarystore.yaml]: Good idea. Go ahead and add it. We'll merge after that.

File: stable/artifactory-ha/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 All changes to this chart will be documented in this file.
 
 ## [0.13.5] - Jun 5, 2019
+* Updated Google Cloud Storage API URL and https settings
```
Comment (eldada) [inline stable/artifactory-ha/CHANGELOG.md]: Need a dedicated new version for your change.

File: stable/artifactory/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,7 @@
 All changes to this chart will be documented in this file.
 
 ## [7.14.4] - Jun 5, 2019
+* Updated Google Cloud Storage API URL and https settings
```
Comment (eldada) [inline stable/artifactory/CHANGELOG.md]: Need a dedicated new version for your change.

File: stable/artifactory-ha/CHANGELOG.md (hunk 3 above)
Comment (matteosilv) [reply stable/artifactory-ha/CHANGELOG.md]: Done. You can check if it looks good

File: stable/artifactory/CHANGELOG.md (hunk 4 above)
Comment (matteosilv) [reply stable/artifactory/CHANGELOG.md]: Done. You can check if it looks good

**Pull request merged**